app/presenters/concerns/hyku_addons/person_or_organization_presenter_behavior.rb

```diff
@@ -81,12 +83,13 @@ def wikidata_link
     private
 
       def person_or_organization_list(field)
-        return [] unless send(field)&.first.present?
+        return if (field_value = solr_document.public_send(field).first.presence || "[]").blank?
 
-        JSON.parse(send(field).first).collect do |hash|
+        JSON.parse(field_value).collect do |hash|
           name = hash.slice("#{field}_family_name", "#{field}_given_name", "#{field}_organization_name").values.map(&:presence).compact.join(', ')
           PersonOrOrganization.new(name, hash["#{field}_orcid"], hash["#{field}_isni"], hash["#{field}_ror"], hash["#{field}_grid"], hash["#{field}_wikidata"])
         end
       end
+      alias participants person_or_organization_list
```

Comment:
**person_or_organization_list** defined elsewhere or it is being defined here for the first time. If so why rename it with alias, while not call it particpants

Reply:
the method `participants` is defined inside of Hyrax Orcid, but some essentialy the same thing as `person_or_organization_list`, so i just aliased it to the existing method here.